src/main/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImpl.java

```diff
@@ -412,6 +412,20 @@ protected void writeExtractCheckFile(PaymentStatus extractedStatus, PaymentProce
         }
     }
 
+    private String createFormattedRemittanceIdTextBasedOnPresenceOfInvoiceDataForDv(String invoiceNumber, Date invoiceDate, String customerPaymentDocumentNumber) {
+        StringBuffer formattedRemittanceIdText = new StringBuffer("");
```

Comment:
I would suggest using a StringBuilder instead for better efficiency. The builder also defaults to being empty, so you don't need to explicitly specify an empty string as the initial value.

Reply:
Changing to StringBuilder as suggested. 
Since the existing code explicitly set the value to an empty string when neither condition was encountered, I would like to retain that for the initialization if you are ok with that.